config/create-common.toml

```diff
@@ -7,7 +7,7 @@
 		#.
 		#.
 		#Prevents all worldgen added by Create from taking effect
-		disableWorldGen = false
+		disableWorldGen = true

```

Comment:
Doesn't this also disable their underground rock formations? Probably better to disable the ores directly by setting their frequency to 0.